### DISCUSSION TITLE
How to Test new Atom Language Package?

### DISCUSSION THREAD

**Discussion by bookofproofs** [atom/atom#22689] [Q&A] [answered]
I'm new to Atom, and I'm trying to develop a new language package based on tree-sitter. Which is the way to go when I want to test my Atom language package before publishing? 

The only way for me to see whether and how my language package works is to publish it first. Only then can I install it in Atom. But this is certainly not the ideal way to do my tests. I wished I could somehow tell Atom to use my local unpublished version of the language package so I could see the behavior of Atom and correct the package before publishing it. 

How do I accomplish this? Or what else are my possibilities to test a new (language, tree-sitter based) package before publishing?

**Comment by icecream17** [comment, marked as answer]
Maybe try the command line `apm install` `apm link`?

If apm is not found:
(Use `apm-beta` if using atom-beta)
(also it's possible that the atom bin folder isn't in path (search add folder to path))

**Reply by bookofproofs** [reply]
I'll try, thank you
